### PR TITLE
Add hint for Team Tic Tac Toe

### PR DESCRIPTION
**Pull request by brad182**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

**Comment by brad182** [comment]
not sure how to fix the <Spoiler> matching error

**Comment by SansPapyrus683** [comment]
oh wow thanks for this
could you uh first check if problems are on the [needed hints page](https://github.com/orgs/cpinitiative/projects/2?pane=issue&itemId=19832146)? i'll still take this, but...

File: solutions/bronze/usaco-831.mdx [hunk 1]
```diff
@@ -183,6 +184,33 @@ public class tttt {
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_tttt_bronze_open18.html)
 
 <LanguageSection>
+
+</Spoiler>
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-831.mdx:188]:
```suggestion
```
should fix

**Comment by bqi343** [comment]
> not sure how to fix the matching error for < Spoiler >

If the spoiler contains `</LanguageSection>` it should also contain `<LanguageSection>`

File: solutions/bronze/usaco-831.mdx [hunk 2]
```diff
@@ -173,15 +172,39 @@ public class tttt {
...
+you check if that pairing is a win?
+
+</Spoiler>
+
+<Spoiler title="Hint 3">
+
+The answer to the previous hint is that the possible pairings are every 
+combination of a letter from A to Z with every other letter. To check 
+if the pairing is a win, review every possible row, column, or diagonal,
+and check if it only contains the two letters.
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-831.mdx:203]: this is just the solution

File: solutions/bronze/usaco-831.mdx [hunk 3]
```diff
@@ -173,15 +172,37 @@ public class tttt {
...
+you check if that pairing is a win?
+
+</Spoiler>
+
+<Spoiler title="Hint 3">
+
+The answer to the previous hint is that the possible pairings are every 
+combination of a letter from A to Z with every other letter. 
+
+</Spoiler>
```
Comment (ryanchou-dev) [inline solutions/bronze/usaco-831.mdx:203]:
```suggestion
<Spoiler title="Hint 2 (answer)">

You can pair up any letter (A...Z) with any other letter.

</Spoiler>
```

**Review by ryanchou-dev — COMMENTED** [review]
HI BRAD SDFSDFA

**Comment by SansPapyrus683** [comment]
hey uh fam there's merge conflicts..

**Comment by bqi343** [comment]
this error needs to be fixed, or else it will be present on all future PRs:

Screen Shot 2023-03-18 at 10 41 48 AM

please wait for pre-commit checks to pass before merging @SansPapyrus683 (alternatively, you can install pre-commit and run the checks locally if you prefer)

**Comment by SansPapyrus683** [comment]
crap, alr then will remember

**Pull request merged**